src/PDPVerifier.sol

```diff
@@ -525,6 +542,7 @@ contract PDPVerifier is Initializable, UUPSUpgradeable, OwnableUpgradeable {
         if (listenerAddr != address(0)) {
             PDPListener(listenerAddr).nextProvingPeriod(setId, nextChallengeEpoch[setId], proofSetLeafCount[setId], extraData);
         }
+        emit NextProvingPeriod(setId, challengeEpoch, proofSetLeafCount[setId]);
```

Comment:
Similar comment applies here - should we include `extraData` in the event payload?